app/components/standup_meeting/check_in_action_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<% if standup_meeting.completed? %>
+  <%= link_to "Edit Response", 
+              edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), 
+              class: 'btn btn-primary' %>
+<% elsif standup_meeting.persisted? %>
+  <%= link_to "Check in",
+              edit_standup_meeting_group_standup_meeting_path(standup_meeting_group, standup_meeting), 
+              class: 'btn btn-primary' %>
+<% else %>
+  <%= button_to "Check in",
```

Comment:
why the same text here?

Reply:
So for one case, the daily standup has already been created (ie via background worker) and this needs to link to the edit page for that existing standup. The other case is that it doesn't exist yet and needs to be created.

These were previously both called 'Check in', and I didn't immediately see a problem with it. I could be down to change them if you have good names for them though.